Address-Book.go

```diff
@@ -27,125 +28,42 @@ type PhoneNum struct {
 	 PhoneNumber []PhoneNum
 
  }
-type User_Contancts struct {
+type UserContacts struct {
 	UserName string
 	Id string
 	Contacts []Contact
 
 }
+type App struct {
+ db *sql.DB
+ err error
+ templates *template.Template
+ User UserContacts
```

Comment:
err, and User members will break this application when multiple users uses it concurrently.